### ISSUE TITLE
Set up the ability for Moodle to send emails

### ISSUE THREAD

**Issue by Pcushing** [WorkHands/moodle#5]
If we can use Sendgrid, great. If not, i'm happy to use another platform that's easier to set up -- SES, Mailgun, etc.

**Comment by erazorbg** [comment]
Does Sendgrid provide SMTP settings that can be used? 

If so Can you provide those settings so I can configure Moodle.

**Comment by Pcushing** [comment]
Yes. I probably wont get to this for a week or so as i'm dealing with a conference this week.
